### DISCUSSION TITLE
How to change the offset for specific map in the beatmaps set?

### DISCUSSION THREAD

**Discussion by nagatsuka-shuuya** [ppy/osu#25308] [Q&A] [answered]
Unfortunately sometimes (for now found only for Graveyard beatmaps) the beatmap set can have different initial offsets for different maps (For example https://osu.ppy.sh/beatmapsets/1871170). 
Looks like there is one shared user offset for beatmap set applied, so I cannot change it for every bitmap individually.
I understand that the actual problem is in the map quality (its absence), but It's problematic to report to the beatmap creator so they can fix the initial beatmap offset in the maps those have wrong offset, so It would be nice to make it possible for the player to fix it locally.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/25299
https://github.com/ppy/osu/issues/24558

**Reply by nagatsuka-shuuya** [reply]
Thank you Bartłomiej
